tests/test_evals.py

```diff
@@ -0,0 +1,48 @@
+import pytest
+
+from bananalyzer.runner.evals import sanitize_string, validate_field_match
+
+
+@pytest.mark.parametrize(
+    "input_str, expected",
+    [
+        ("hello world!", "helloworld"),
+        ("HELLO_WORLD", "helloworld"),
+        ("Hello1 WoRlD@!", "hello1world"),
+        ("", ""),
+        ("123456", "123456"),
+        ("!@#$%^&*()", ""),
```

Comment:
This doesn't seem right lol

I thought we were only going to allow for 2 alphanumeric changes